### ISSUE TITLE
Fix frontend-platform draft release issue

### ISSUE THREAD

**Issue by davidjoy** [openedx/frontend-wg#34]
Today whenever frontend-platform creates a new release on Github, the release goes into a draft state for some reason and is only visible to admins.

This is keeping folks from seeing the latest releases of frontend-platform, causing confusion.

In a recent thread on a semantic-release issue, one of the maintainers posited why frontend-platform is having this problem: https://github.com/semantic-release/github/issues/275#issuecomment-841424337

tl;dr, we should zip the artifact that we create in Github. It may be failing to finish the release because it's taking too long to upload all the files.

**Comment by davidjoy** [comment]
This seems to have righted itself!  Going to close this until I see it happen again.
